aggregator/src/param.rs

```diff
@@ -20,8 +20,8 @@ impl ConfigParams {
         Self {
             strategy: FpStrategy::Simple,
             degree: 19,
-            num_advice: vec![75],
-            num_lookup_advice: vec![10],
+            num_advice: vec![100],
+            num_lookup_advice: vec![13],
```

Comment:
if we still use chunk num = 15, we don't need to increase nums here?

Reply:
I am looking into what changes caused the increased column usage. The params were set to their previous values here: increased https://github.com/scroll-tech/zkevm-circuits/pull/1225/commits/edb0ae70e6dae4914d80387da51494effcf195d7